submissions/motunrayoIlawole/Database/init.sql

```diff
@@ -0,0 +1,89 @@
+CREATE DATABASE Top_clothing;
```

Comment:
You should always use all lowercase when it comes to database. I think the name should all be in lowercase. 
Everything else looks good. Well done. @TunrayoIlawole